editor/osm_auth_tests/osm_auth_tests.cpp

```diff
@@ -18,29 +16,23 @@ static constexpr char const * kForgotPasswordEmail = "osmtest1@organicmaps.app";
 
 UNIT_TEST(OSM_Auth_InvalidLogin)
 {
-  testing::RunOnNetworkThread([]
-  {
-    OsmOAuth auth = OsmOAuth::DevServerAuth();
-    bool result;
-    TEST_NO_THROW(result = auth.AuthorizePassword(kValidOsmUser, kInvalidOsmPassword), ());
-    TEST_EQUAL(result, false, ("invalid password"));
-    TEST(!auth.IsAuthorized(), ("Should not be authorized."));
-  });
+  OsmOAuth auth = OsmOAuth::DevServerAuth();
```

Comment:
```suggestion
  OsmOAuth auth = OsmOAuth::ServerAuth();
```

Reply:
@strump 

```
Running osm_auth_tests.cpp::OSM_Auth_Login
FAILED
osm_auth_tests/osm_auth_tests.cpp:30 Unexpected exception at TEST(result = auth.AuthorizePassword(kValidOsmUser, kValidOsmPassword)) HTTP 401
```

Can you please help to debug what is wrong? We need some "test" credentials for the prod.